source/AI.cpp

```diff
@@ -472,7 +530,7 @@ void AI::Step(const PlayerInfo &player, Command &activeCommands)
 		double healthRemaining = it->Health();
 		bool isPresent = (it->GetSystem() == playerSystem);
 		bool isStranded = IsStranded(*it);
-		bool thisIsLaunching = (isLaunching && isPresent);
+		bool thisIsLaunching = (isPresent && IsLaunching(*it));
```

Comment:
Are there other uses of `thisIsLaunching` that need to be revisited, now that launching is a selective process for player escorts?

Reply:
It is used 3 times in the code after this point, and all 3 locations seem to make sense in the new situation;
- line 540: Disabled ships under the players command only deploy when the player commands the carried ships to deploy.
- line 567: Intented deploy behavior, only ships with carried ships that deploy will deploy.
- line 835: Recall behavior, player ships only recall when the carrier and carried ship are in a different system or when the player explicitly orders a recall.